parachain-system/src/lib.rs

```diff
@@ -96,6 +106,19 @@ decl_storage! {
 		/// This data is also absent from the genesis.
 		HostConfiguration get(fn host_configuration): Option<AbridgedHostConfiguration>;
 
+		/// The last downward message queue chain head we have observed or all zeroes if we haven't observed
+		/// any.
+		///
+		/// This value is loaded before and saved after processing inbound downward messages carried
+		/// by the system inherent.
+		LastDmqMqcHead: relay_chain::Hash;
```

Comment:
I didn't do this change, because I changed `relay_chain::Hash` to `MessageQueueChain`, which hopefully makes it clear that we rely on the `Default` implementation here.

I am not fundamentally opposed to change it to `Option` though, so just tell